### PR TITLE
[KOGITO-1214] - Do not use echo statement on the images to print log

### PR DESCRIPTION
**Pull request by vaibhavjainwiz**
Jira Issue : https://issues.redhat.com/browse/KOGITO-1214

### CODE REVIEW

File: modules/kogito-graalvm-scripts/added/memory-limit.sh [hunk 1]
```diff
@@ -11,22 +11,25 @@
...
+        log_info "Limit memory for this container is set to ${LIMIT_MEMORY}. Allocated memory for JVM will be set to ${jvm_limit_memory}."
         if [ "${jvm_limit_memory}" -lt "${limit}" ]; then
             limit=$(echo "scale=1; ${limit} / (80/100)" | bc -l)
             printf "Provided memory (${LIMIT_MEMORY}) limit is too small (should be greater then %.0f bytes), native build will use all available memory.\n" $limit
         else
             export KOGITO_OPTS="${KOGITO_OPTS} -Dnative-image.xmx=${jvm_limit_memory}"
         fi
     else
-        echo "Provided memory (${LIMIT_MEMORY}) limit is not valid, native build will use all available memory"
+        log_info "Provided memory (${LIMIT_MEMORY}) limit is not valid, native build will use all available memory"
```
Comment (spolti) [inline modules/kogito-graalvm-scripts/added/memory-limit.sh]: set it as warn.

**Review by spolti — CHANGES_REQUESTED** [review]
Nice work. just one comment.

File: modules/kogito-graalvm-scripts/added/memory-limit.sh (hunk 1 above)
Comment (vaibhavjainwiz) [reply modules/kogito-graalvm-scripts/added/memory-limit.sh]: done

**Review by sutaakar — APPROVED** [review]
Looks good to me

**Review by spolti — APPROVED** [review]

**Comment by spolti** [comment]
@radtriste do you mind to take a look?

**Review by radtriste — APPROVED** [review]

**Pull request merged**